kvui.py

```diff
@@ -541,9 +542,9 @@ def on_press(text):
 
 
 status_icons = {
-    HintStatus.HINT_NO_PRIORITY: "information",
-    HintStatus.HINT_PRIORITY: "exclamation-thick",
-    HintStatus.HINT_AVOID: "alert"
+    HintStatus.HINT_PRIORITY_NO_PRIORITY: "information",
+    HintStatus.HINT_PRIORITY_PRIORITY: "exclamation-thick",
+    HintStatus.HINT_PRIORITY_AVOID: "alert"
 }
 
```

Comment:
Leaving this as a note:
I moved everything else like statuc_colors etc. to NetUtils, but not this, which might feel a bit awkward

I left this here because while colors and display order are more universal, these icons are very kivymd-specific and not useful to anyone else I think

Happy to hear opinions on this